android/src/main/java/com/RNFetchBlob/RNFetchBlobFS.java

```diff
@@ -665,10 +665,14 @@ static void exists(String path, Callback callback) {
             }
         }
         else {
-            path = normalizePath(path);
-            boolean exist = new File(path).exists();
-            boolean isDir = new File(path).isDirectory();
-            callback.invoke(exist, isDir);
+            try {
+                path = normalizePath(path);
+                boolean exist = new File(path).exists();
+                boolean isDir = new File(path).isDirectory();
+                callback.invoke(exist, isDir);
+            } catch (NullPointerException e) {
+                callback.invoke(false, false);
```

Comment:
It is not that the path does not exist; it is that the "path" was actually not a path. We should be rejecting the promise.